src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/applications/claimant/ClaimantTellSomethingElseController.java

```diff
@@ -60,7 +60,7 @@ public ResponseEntity<CCDCallbackResponse> aboutToStartClaimantTSE(
             @RequestBody CCDRequest ccdRequest) {
 
         CaseData caseData = ccdRequest.getCaseDetails().getCaseData();
-        if (Boolean.FALSE.equals(Helper.isRespondentSystemUser(caseData))) {
+        if (!Helper.isRespondentSystemUser(caseData)) {
```

Comment:
Rather than checking for false, we could consider updating the logic to explicitly use the Helper.isRespondentSystemUser(caseData) condition, like so:
if (Helper.isRespondentSystemUser(caseData)) {
    caseData.setClaimantTseRespNotAvailable(NO);
} else {
    caseData.setClaimantTseRespNotAvailable(YES);
}
This approach enhances clarity by directly expressing the intent and may improve readability for future maintainers.